src/data-dictionary/events/K8sContainerSample/containerMemoryMappedFileBytes.md

```diff
@@ -0,0 +1,9 @@
+---
+name: containerMemoryMappedFileBytes
+type: attribute
+units:
```

Comment:
there's a few that has the specific unit in the description but lacking it in the 'units' field

Reply:
This is corrected now.